bpf/process/bpf_process_event.h

```diff
@@ -166,6 +166,7 @@ struct cwd_read_data {
 	struct dentry *dentry;
 	struct vfsmount *vfsmnt;
 	struct mount *mnt;
+	int error;
```

Comment:
nit: I believe error can be a local variable in the new function, since we seem to always check for ->resolved in the end and set the error to UNRESOLVED_PATH_COMPONENTS.

Reply:
sounds good, will check, thanks